srtcore/api.h

```diff
@@ -276,7 +281,7 @@ friend class CRendezvousQueue;
 private:
    volatile bool m_bClosing;
    srt::sync::Mutex m_GCStopLock;
-   pthread_cond_t m_GCStopCond;
+   srt::sync::CCondVar m_GCStopCond;
```

Comment:
This will not work, at least on Windows. m_GCStopCond needs to be initialized with `PTHREAD_COND_INITIALIZER` or allocated dynamically. CUDTUnited has a static instance, so the initialization of this CV should not be done in its constructor.